gloss/Graphics/Gloss/Geometry/Triangulation.hs

```diff
@@ -0,0 +1,388 @@
+module Graphics.Gloss.Geometry.Triangulation
```

Comment:
Please add a comment describing where a maintainer could read more about the algorithm implemented in this module. If we find a bug in this code we'll need to read the original paper to understand how the algorithm works.